apiserver/httpcontext/model_test.go

```diff
@@ -46,18 +49,19 @@ func (s *ModelHandlersSuite) SetUpTest(c *gc.C) {
 	}
 	mux := apiserverhttp.NewMux()
 	mux.AddHandler("GET", "/query", s.queryHandler)
-	mux.AddHandler("GET", "/implied", s.impliedHandler)
```

Comment:
Just to check, this is just a change in tests right? We're not changing the name of an actual api endpoint

Reply:
Yeah, 100% only tests